README.md

```diff
@@ -19,6 +19,11 @@ and [MQTT Cover](https://www.home-assistant.io/integrations/cover.mqtt/) platfor
 $ pip3 install --user --upgrade switchbot-mqtt
 ```
 
+**NOTE**: On Raspberry Pi you also need `libglib2.0-dev`. Install it with 
+```sh
+$ apt install libglib2.0-dev -y
```

Comment:
`apt-get` is redundant, `apt` does the same thing. also `$` is the convention for `sudo`

Reply:
`apt` is a higher-level tool than `apt-get`, I would use the "more direct" variant (i.e. `apt-get`), when the features of `apt` are not required.

by convention, prefix `$` typically refers a non-privileged shell; prefix `#` to a privileged shell. anyway, I would prefer the more explicit version with `sudo` (for easier copy and pasting of the user has not switched to the root account yet)

Reply:
for `$` (non-privileged) vs. `#` (privileged) see e.g. bash's man page:
```
PROMPTING
[...]
\$     if the effective UID is 0, a #, otherwise a $
```